identity-apps-core/apps/authentication-portal/src/main/webapp/fido2-identifierfirst.jsp

```diff
@@ -0,0 +1,212 @@
+    } else {
+        // Resolve tenant domain for the authentication API URL.
+        authAPIURL = AuthenticationEndpointUtil.resolveTenantDomain(authAPIURL);
+    }
+    if (!authAPIURL.endsWith("/")) {
+        authAPIURL += "/";
+    }
+    authAPIURL += "context/" + request.getParameter("sessionDataKey");
+    String contextProperties = AuthContextAPIClient.getContextProperties(authAPIURL);
+    Gson gson = new Gson();
+    Map data = gson.fromJson(contextProperties, Map.class);
+    
+    boolean enablePasskeyProgressiveEnrollment = (boolean) data.get("FIDO.EnablePasskeyProgressiveEnrollment");
+
+%>
+
+<%-- Branding Preferences --%>
+<jsp:directive.include file="includes/branding-preferences.jsp" />
+
+<!doctype html>
+<html lang="en-US">
+<head>
+    <%-- header --%>
+    <%
+        File headerFile = new File(getServletContext().getRealPath("extensions/header.jsp"));
+        if (headerFile.exists()) {
+    %>
+    <jsp:include page="extensions/header.jsp"/>
+    <% } else { %>
+    <jsp:include page="includes/header.jsp"/>
+    <% } %>
+
+    <%-- analytics --%>
+    <%
+        File analyticsFile = new File(getServletContext().getRealPath("extensions/analytics.jsp"));
+        if (analyticsFile.exists()) {
+    %>
+        <jsp:include page="extensions/analytics.jsp"/>
+    <% } else { %>
+        <jsp:include page="includes/analytics.jsp"/>
+    <% } %>
+</head>
+<body class="login-portal layout authentication-portal-layout">
+    <% if (new File(getServletContext().getRealPath("extensions/timeout.jsp")).exists()) { %>
+        <jsp:include page="extensions/timeout.jsp"/>
+    <% } else { %>
+        <jsp:include page="util/timeout.jsp"/>
+    <% } %>
+
+    <layout:main layoutName="<%= layout %>" layoutFileRelativePath="<%= layoutFileRelativePath %>" data="<%= layoutData %>" >
+        <layout:component componentName="ProductHeader">
+            <%-- product-title --%>
+            <%
+                File productTitleFile = new File(getServletContext().getRealPath("extensions/product-title.jsp"));
+                if (productTitleFile.exists()) {
+            %>
+                <%
+                    if (StringUtils.equals(tenantForTheming, IdentityManagementEndpointConstants.SUPER_TENANT)) {
+                %>
+                    <div class="product-title">
+                        <jsp:include page="extensions/product-title.jsp"/>
+                    </div>
+                <% } else { %>
+                    <jsp:include page="extensions/product-title.jsp"/>
+                <% } %>
+            <% } else { %>
+                <jsp:include page="includes/product-title.jsp"/>
+            <% } %>
+        </layout:component>
+        <layout:component componentName="MainSection">
+            <div class="ui segment">
+                <h3 class="ui header ellipsis">
+                    <span id="fido-identifier-header">
+                        <%=AuthenticationEndpointUtil.i18n(resourceBundle, "fido.title.login" )%>
```

Comment:
Please check the other places as well.

```suggestion
                        <%=AuthenticationEndpointUtil.i18n(resourceBundle, "fido.title.login")%>
```

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/4156/commits/8968e9e9e80de6f997577d12f3f6222db508b1de